CODEOWNERS

```
@@ -225,8 +225,8 @@ k8s/**/common/xui/register-org-ingress.yaml @hmcts/xui
 k8s/perftest/common/xui/share-case.yaml @hmcts/xui
 
 ### PCQ
-k8s/aat/common/pcq/namespace.yaml @hmcts/pcq-admins
-k8s/aat/common/pcq/identity.yaml @hmcts/pcq-admins
+k8s/**/common/pcq/namespace.yaml @hmcts/pcq-admins
+k8s/**/common/pcq/identity.yaml @hmcts/pcq-admins
```

you aren't prod approved yet so can't have **, the only restricted environments are AAT and prod, so you don't need to change CODEOWNERS

If you want it to your team gets automatically requested for review you could add:

```suggestion
k8s/demo/common/pcq/*.yaml @hmcts/pcq-admins
```

---

Do we not need the line for AAT? `k8s/aat/common/pcq/*.yaml @hmcts/pcq-admins`

---

no aat wildcard isn't allowed each app has to be checked, since your CODEOWNERS change is only for namespace and identity then its fine